Misc/NEWS.d/next/Core and Builtins/2019-08-09-18-28-57.bpo-37802.pKxcAW.rst

```diff
@@ -0,0 +1,2 @@
+Slightly improve perfomance of PyLong_FromSize_t(). Patch by Sergey
```

Comment:
Thanks for the PR.

I would recommend adding reST inline markup to "PyLong_FromSize_t". After Sphinx processes the news entry, it allows users to click on the function to be directed to the relevant documentation.
```suggestion
Slightly improve perfomance of :c:func:`PyLong_FromSize_t`. Patch by Sergey
```